plugins/linux-capture/pipewire.c

```diff
@@ -83,6 +97,13 @@ struct _obs_pipewire_data {
 	struct pw_core *core;
 	struct spa_hook core_listener;
 
+	// version of the connected PipeWire server
+	struct pw_version server_version;
+	// version of the linked PipeWire library
+	struct pw_version library_version;
+	// version of the PipeWire headers at compile time
+	struct pw_version header_version;
```

Comment:
I don't think we care about `library_version` and `header_version` for now, nor that we will care about them. We can bump the minimum PipeWire version to 0.3.40, and update the Flatpak package. (Distros that don't support a recent enough version of PipeWire wouldn't be able to update OBS Studio, but that's **precisely** why I encourage using the Flatpak build!)